src/Modal/README.md

```diff
@@ -16,7 +16,7 @@ Provides a basic modal component with customizable title, body, and footer butto
 ### `buttons` (element or shape in form of buttonPropTypes array; optional)
 `buttons` is an array of either elements or shapes that take the form of the buttonPropTypes. See the [buttonPropTypes](https://github.com/edx/paragon/blob/master/src/Button/index.jsx#L40) for a list of acceptable props to pass as part of a button.
 
-### `closeText` (string; optional)
+### `closeText` (string or element; optional)
```

Comment:
I just want to point out that I don't think a prop named `closeText` should take an `element` as the prop name would suggest only a `string`.

I don't think it's worth a breaking change at this point, but I think it's useful to add to the next major upgrade branch / issues.

Reply:
Yeah, I didn't make that change because it would be a breaking change and we'd rather have the fix in SFE sooner rather than later. I'll mention it in the issue though.